core/src/services/web-components.js

```diff
@@ -19,7 +19,7 @@ class WebComponentSvcClass {
    *  @property {object} extendedContext
    *  @property {object} node
    *  @property {string} nodeId
-   *  @property {boolean} isSpecialMf
+   *  @property {boolean} isSpecialMf - allows you to specify whether the web component is rendered in a modal, splitView or drawer (`false` by default)

```

Comment:
```suggestion
   *  @property {boolean} isSpecialMf allows you to specify whether the web component is rendered in a modal, splitView or drawer (`false` by default)
```
I would remove the minus sign.